server/app/views/admin/ti/TrustedIntermediaryGroupListView.java

```diff
@@ -110,26 +140,31 @@ private DivTag renderAddNewButton(Http.Request request) {
   private TrTag renderGroupRow(TrustedIntermediaryGroupModel ti, Http.Request request) {
     return tr().withClasses(
             ReferenceClasses.ADMIN_TI_GROUP_ROW,
+            "cf-ti-element",
             "border-b",
             "border-gray-300",
             StyleUtils.even("bg-gray-100"))
         .with(renderInfoCell(ti))
         .with(renderMemberCountCell(ti))
+        .withData(QuestionSortOption.TI_NAME.getDataAttribute(), ti.getName())
+        .withData(
+            QuestionSortOption.TI_NUM_MEMBERS.getDataAttribute(),
+            Integer.toString(ti.getTrustedIntermediaries().size()))
         .with(renderActionsCell(ti, request));
   }
 
   private TdTag renderInfoCell(TrustedIntermediaryGroupModel tiGroup) {
     return td().with(div(tiGroup.getName()).withClasses("font-semibold"))
         .with(div(tiGroup.getDescription()).withClasses("text-xs"))
-        .withClasses(BaseStyles.TABLE_CELL_STYLES, "pr-12");
+        .withClasses(BaseStyles.TABLE_CELL_STYLES, "pr-12", "cf-ti-info");
```

Comment:
suggestion (blocking): since `cf-ti-info` is only used in tests, add it as a `data-testid` attribute. then you can use the `getByTestId` selector to select it directly. same for `cf-ti-member` below.

Example:
https://github.com/civiform/civiform/blob/6c96e1dd9ddd7a7b2f5f93dfc9d9620c1ac6dac5/server/app/views/applicant/TrustedIntermediaryClientListView.java#L182

https://github.com/civiform/civiform/blob/6c96e1dd9ddd7a7b2f5f93dfc9d9620c1ac6dac5/browser-test/src/trusted_intermediary.test.ts#L613

Also, double check if you should use `cf` for classes that are only used in tests. I think we generally use them as reference classes for js functionality, but tbh I'm not totally clear on the guidelines. We should talk about it as a group!

Reply:
i don't see the cf- prefix used for other test ids so I'll remove it